Formula/cdo.rb

```diff
@@ -0,0 +1,30 @@
+class Cdo < Formula
+  desc "Climate Data Operators"
+  homepage "https://code.mpimet.mpg.de/projects/cdo"
+  url "https://code.mpimet.mpg.de/attachments/download/20826/cdo-1.9.8.tar.gz"
+  sha256 "f2660ac6f8bf3fa071cf2a3a196b3ec75ad007deb3a782455e80f28680c5252a"
+
+  depends_on "eccodes"
+  depends_on "hdf5"
+  depends_on "netcdf"
+  depends_on "proj"
```

Comment:
Is this a runtime requirement?

Reply:
Yes, this is a conversion and manipulation utility to slice and dice meteorological numerical data, and these come in various formats. This utility links against the mentioned libraries to be able to convert between competing formats, e.g. netcdf to GRIB and vice versa.